codespell_lib/data/dictionary.txt

```diff
@@ -692,14 +862,18 @@ approxiamtion->approximation
 approxiately->approximately
 approximatly->approximately
 approximitely->approximately
+appy->apply
 aprehensive->apprehensive
+aproach->approach
 apropriate->appropriate
 apropriately->appropriately
 aproval->approval
 aproximate->approximate
 aproximately->approximately
 aproximation->approximation
 aproximations->approximations
+aprrovement->approvement
```

Comment:
The word approvement is archaic, it isn't known by my the Firefox spell checker. I feel like we shouldn't be encouraging programmers to use it. Approval is a better choice of word.

Reply:
Good point. Changed